mainnet-1/specs/movement.json

```diff
@@ -0,0 +1,629 @@
+{
+    "proposal": {
+        "title": "Add Specs: Movement",
+        "description": "Adding new specification support for relaying Movement data on Lava",
+        "specs": [
+            {
+                "index": "MOVEMENT",
+                "name": "movement mainnet",
+                "enabled": true,
+                "reliability_threshold": 268435455,
+                "data_reliability_enabled": true,
+                "block_distance_for_finalized_data": 0,
+                "blocks_in_finalization_proof": 5,
+                "average_block_time": 200,
```

Comment:
From their explorer it seems movement is slightly slower than Aptos. 
https://explorer.movementnetwork.xyz/blocks?network=testnet
seems like average block time is around 1 second.

Reply:
Changed to 1s